lib/serializer.ex

```diff
@@ -0,0 +1,70 @@
+defmodule Membrane.Element.IVF.Serializer do
+  @moduledoc false
+  use Membrane.Filter
+  use Membrane.Log
+
+  alias Membrane.Element.IVF
+  alias Membrane.{Buffer, RemoteStream}
+  alias Membrane.Caps.VP9
+  alias Membrane.Caps.VP8
```

Comment:
we don't have VP8 nor VP9 caps, and when we do, they will be called `Membrane.VPx`